core/src/main/scala/filodb.core/memstore/IndexBootstrapper.scala

```diff
@@ -41,6 +41,7 @@ class IndexBootstrapper(colStore: ColumnStore) {
       }
       .countL
       .map { count =>
+        index.commit()
```

Comment:
I am not sure ``commit`` will help. While commit ensures readers see the latest data, refreshing the index (next line of code) will achieve that too. We do not care about durability of index yet, which is what `` commit `` guarantees.